Dockerfile_jupyterlab

```diff
@@ -0,0 +1,191 @@
+    && tar -xvf parmetis_4.0.3.orig.tar.gz \
+    && cd parmetis-4.0.3 \
+    && sed -i 's/IDXTYPEWIDTH 32/IDXTYPEWIDTH 64/g' ./metis/include/metis.h \
+    && CC=mpicc CXX=mpicxx make config prefix=/home/tools \
+    && make -j$(nproc) && make install \
+    && if [ "$TARGETARCH" == "arm64" ]; \
+       then cp build/Linux-aarch64/libmetis/libmetis.a /home/tools/lib; \
+       else cp build/Linux-x86_64/libmetis/libmetis.a /home/tools/lib; fi \
+    && cp metis/include/metis.h /home/tools/include
+
+RUN wget --progress=bar:force:noscroll https://www.lua.org/ftp/lua-5.3.6.tar.gz \
+    && tar -xzvf lua-5.3.6.tar.gz \
+    && cd lua-5.3.6 && make linux CC=mpicc && make local \
+    && cp -r install/* /home/tools && cd ..
+
+RUN wget --progress=bar:force:noscroll https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.gz \
+    && tar -xf eigen-3.4.0.tar.gz \
+    && cd eigen-3.4.0 && mkdir build && cd build && cmake .. -DCMAKE_INSTALL_PREFIX=/home/tools \
+    && make -j$(nproc) install 
+
+RUN git clone --depth 1 --single-branch --branch 4.9.3 https://github.com/OSGeo/PROJ.git \
+    && cd PROJ \
+    && mkdir build && cd build \
+    && CC=mpicc CXX=mpicxx cmake .. -DCMAKE_INSTALL_PREFIX=/home/tools \
+    && make -j$(nproc) && make install
+
+### Put all dependencies, which point to a specific version, before this comment
+### Put all dependencies, which use the latest version, after this comment to reduce build time
+
+RUN git clone --recursive https://github.com/TUM-I5/ASAGI.git \
+    && cd ASAGI \
+    && mkdir build && cd build \
+    && CC=mpicc CXX=mpicxx cmake .. -DCMAKE_INSTALL_PREFIX=/home/tools -DSHARED_LIB=off -DSTATIC_LIB=on -DNONUMA=on \
+    && make -j$(nproc) && make install
+
+RUN git clone https://github.com/SeisSol/easi \
+    && cd easi \
+    && mkdir build && cd build \
+    && CC=mpicc CXX=mpicxx cmake .. -DEASICUBE=OFF -DLUA=ON -DCMAKE_PREFIX_PATH=/home/tools -DCMAKE_INSTALL_PREFIX=/home/tools -DASAGI=ON -DIMPALAJIT=OFF .. \
+    && make -j$(nproc) && make install
+
+RUN pip install numpy && pip install git+https://github.com/SeisSol/PSpaMM.git@davschneller/compile-fixes && docker-clean
+
+#RUN git clone --depth 1 --single-branch --branch main https://github.com/libxsmm/libxsmm.git \
+#    && cd libxsmm \
+#    && make PLATFORM=1 JIT=1 AR=aarch64-linux-gnu-ar \
+#       FC=aarch64-linux-gnu-gfortran \
+#       CXX=aarch64-linux-gnu-g++ \
+#       CC=aarch64-linux-gnu-gcc \
+#       -j$(nproc) generator \
+#    && cp bin/libxsmm_gemm_generator /home/tools/bin
+
+RUN git clone --recursive --depth 1 --single-branch --branch v1.1.3 https://github.com/SeisSol/SeisSol.git \
+    && cd SeisSol \
+    && mkdir build_hsw && cd build_hsw \
+    && export PATH=$PATH:/home/tools/bin \
+    && CC=mpicc CXX=mpicxx cmake .. -DCMAKE_PREFIX_PATH=/home/tools -DGEMM_TOOLS_LIST=PSpaMM -DHOST_ARCH=noarch -DASAGI=on -DNETCDF=on -DORDER=4 \
```

Comment:
`-DDR_QUAD_RULE_OPTIONS=dunavant`
if we want to harmonize the dockerfiles and decrease run-times.